### DISCUSSION TITLE
relatedRecordsList - customization

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16588] [How to]
How to customize table generated via relatedRecordsList without mending core?

**Comment by apcloic** [comment]
You can do it with a custom Textparser script :
https://github.com/YetiForceCompany/YetiForceCRM/issues/13278#issuecomment-711648335

**Comment by rskrzypczak** [comment]
Use the RelatedRecordsData (_Related records data (array)_) function and the twig function instead
![image](https://user-images.githubusercontent.com/10221999/196082910-ca872322-f43e-452d-84a1-18541a01bca5.png)


`YFParser('$(custom : RelatedRecordsData|__RELATION_MODULE_OR_RELATION_ID__|__FIELDS__|__CONDITIONS__|__LIMIT__|__ORDER_BY__|__RELATION_CONDITION__)$')`

![image](https://user-images.githubusercontent.com/10221999/196083351-49cc9d17-acf6-4e6d-a558-6c9f8514f129.png)
